opm/material/fluidmatrixinteractions/EclEpsConfig.hpp

```diff
@@ -196,8 +196,9 @@ class EclEpsConfig
                       const Opm::EclipseState& eclState,
                       Opm::EclTwoPhaseSystemType twoPhaseSystemType)
     {
+        const auto& endscale = eclState.runspec().endpoint_scaling();
```

Comment:
> Nothing is consistent in parse

that's a new function isn't it?

> but that'll hopefully be addressed soon.

I hope this is done in a way which makes the conventions used by opm-parser *more* similar to the other modules, not *less*!?